airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -50,42 +46,44 @@ class VaultBackend(BaseSecretsBackend, LoggingMixin):
     :type url: str
-    :param auth_type: Authentication Type for Vault (one of 'token', 'ldap', 'userpass', 'approle',
-        'github', 'gcp', 'kubernetes'). Default is ``token``.
+    :param auth_type: Authentication Type for Vault. Default is ``token``. Available values are in
+        :py:const:`airflow.providers.hashicorp.hooks.vault.VALID_AUTH_TYPES`.
     :type auth_type: str
-    :param mount_point: The "path" the secret engine was mounted on. (Default: ``secret``)
+    :param mount_point: The "path" the secret engine was mounted on. Default is "secret". Note that
+         this mount_point is not used for authentication if authentication is done via a
+         different engine.
     :type mount_point: str
+    :param kv_engine_version: Select the version of the engine to run (``1`` or ``2``, default: ``2``).
+    :type kv_engine_version: int
     :param token: Authentication token to include in requests sent to Vault.
         (for ``token`` and ``github`` auth_type)
     :type token: str
-    :param kv_engine_version: Select the version of the engine to run (``1`` or ``2``, default: ``2``)
-    :type kv_engine_version: int
-    :param username: Username for Authentication (for ``ldap`` and ``userpass`` auth_type)
+    :param username: Username for Authentication (for ``ldap`` and ``userpass`` auth_type).
     :type username: str
-    :param password: Password for Authentication (for ``ldap`` and ``userpass`` auth_type)
+    :param password: Password for Authentication (for ``ldap`` and ``userpass`` auth_type).
     :type password: str
-    :param role_id: Role ID for Authentication (for ``approle`` auth_type)
+    :param secret_id: Secret ID for Authentication (for ``approle`` auth_type).
+    :type secret_id: str
+    :param role_id: Role ID for Authentication (for ``approle`` auth_type).
     :type role_id: str
-    :param kubernetes_role: Role for Authentication (for ``kubernetes`` auth_type)
+    :param kubernetes_role: Role for Authentication (for ``kubernetes`` auth_type).
     :type kubernetes_role: str
-    :param kubernetes_jwt_path: Path for kubernetes jwt token (for ``kubernetes`` auth_type, deafult:
-        ``/var/run/secrets/kubernetes.io/serviceaccount/token``)
+    :param kubernetes_jwt_path: Path for kubernetes jwt token (for ``kubernetes`` auth_type, default:
+        ``/var/run/secrets/kubernetes.io/serviceaccount/token``).
     :type kubernetes_jwt_path: str
-    :param secret_id: Secret ID for Authentication (for ``approle`` auth_type)
-    :type secret_id: str
-    :param gcp_key_path: Path to GCP Credential JSON file (for ``gcp`` auth_type)
+    :param gcp_key_path: Path to GCP Credential JSON file (for ``gcp`` auth_type).
     :type gcp_key_path: str
-    :param gcp_scopes: Comma-separated string containing GCP scopes (for ``gcp`` auth_type)
+    :param gcp_scopes: Comma-separated string containing GCP scopes (for ``gcp`` auth_type).
     :type gcp_scopes: str
-    """
+   """
```

Comment:
This looks like an errant delete happened?

```suggestion
    """
```